docs/tutorials/add-catalog.md

```diff
@@ -35,7 +35,7 @@ This catalog is distributed as an image [quay.io/operatorhubio/catalog](https://
         type: Image
         image:
           ref: <catalog_image>
-          pollInterval: <poll_interval_duration>
+          PollIntervalMinutes: <poll_interval_duration>
```

Comment:
Another one

Reply:
The first letter should not be capitalized here, right?